api/api/routers/mcp/mcp_server.py

```diff
@@ -252,26 +252,31 @@ async def fetch_run_details(
 
 @_mcp.tool()
 async def get_agent_versions(
-    task_id: Annotated[str, Field(description="The task ID of the agent")],
+    # why is this tool not using the agent_id?
+    agent_id: Annotated[str, Field(description="The ID of the agent")],
     version_id: Annotated[
         str | None,
         Field(description="An optional version id, e-g 1.1. If not provided all versions are returned"),
     ] = None,
 ) -> MCPToolReturn:
     """<when_to_use>
     When the user wants to retrieve details of versions of a WorkflowAI agent, or when they want to compare a specific version of an agent.
+
```

Comment:
I think this would be a good place to remain the MCP client: what is an agent version and how does it differ from a "schema ID".